package.json

```diff
@@ -84,5 +86,8 @@
   },
   "optionalDependencies": {
     "node-sass": "^4.12.0"
-  }
+  },
+  "browserslist": [
```

Comment:
btw I've been trending toward providing .rc files for a lot of our tools but I don't have a specific reason for doing this. Do you have any reasons why you prefer putting stuff into package.json versus .rc? I guess one reason is that it's less files to keep track of?

Reply:
I don't honestly have a strong opinion. Probably fewer files is ideal.

Reply:
ok. we can add this to package.json and think about if we want to clean up the other .rc files in a future PR